arbitrum-docs/how-arbitrum-works/06-optimistic-rollup.mdx

```diff
@@ -43,13 +55,13 @@ The alternative to interactive proving would be to have a rollup block contain a
 
 We believe strongly that interactive proving is the superior approach, for the following reasons.
 
-**More efficient in the optimistic case**: Because interactive proving can resolve disputes that are larger than one transaction, it can allow a rollup block to contain only a single claim about the end state of the chain after all of the execution covered by the block. By contrast, reexecution requires posting a state claim for each transaction within the rollup block. With hundred or thousands of transactions per rollup block, this is a substantial difference in L1 footprint -- and L1 footprint is the main component of cost.
+**More efficient in the optimistic case**: Because interactive proving can resolve disputes that are larger than one transaction, it can allow a rollup block to contain only a single claim about the end state of the chain after all of the execution covered by the block. By contrast, re-execution requires posting a state claim for each transaction within the rollup block. With hundreds or thousands of transactions per rollup block, this is a substantial difference in L1 footprint -- and L1 footprint is the main component of cost of operating an L2.
```

Comment:
```suggestion
**More efficient in the optimistic case**: Because interactive proving can resolve disputes larger than one transaction, it can allow a Rollup block to contain only a single claim about the end state of the chain after all of the execution covered by the block. By contrast, re-execution requires posting a state claim for each transaction within the rollup block. With hundreds or thousands of transactions per rollup block, there is a substantial difference in L1 footprint, and the L1 footprint is the main component of the cost of operating an L2.
```